config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
```

Comment:
I don't really like having inaccurate configuration in the project -- I would assume that a production deployment would fail, because we don't have a Redis instance. However, since we've tested the branch in dev already, I'm going to assume that it works as-is.